code/datums/beam.dm

```diff
@@ -128,9 +132,13 @@
 	owner = null
 	return ..()
 
+/obj/effect/ebeam/New(alpha_fade = 0, time_exist = 10)
+	if(alpha_fade)
+		animate(src, alpha = 0, time = time_exist*10)
```

Comment:
Also this is going to look like crap if the beam moves, because moving will interrupt the fade.